doc/03-Configuration.md

```diff
@@ -34,17 +29,37 @@ port:
 
 `443,5665-5669`
 
-Scan jobs can be executed using the `icingacli x509 scan` CLI command. The `--job` option is used to specify the scan
-job which should be run:
+Additionally, each job may also exclude specific **hosts** and **IP** addresses from scan. These hosts won't be scanned
+when you run the [scan](04-Scanning.md#scan-command) or [jobs](04-Scanning.md#scheduling-jobs) command. Excluding an
+entire network and specifying IP addresses in CIDR format will not work. You must specify a comma-separated concrete
+**IP** and **host CN**, e.g:
 
-```
-icingacli x509 scan --job lan
-```
+`192.0.2.2,192.0.2.5,icinga.com`
+
+### Job Schedules
+
+Schedules are `cron` and [rrule](https://www.rfc-editor.org/rfc/rfc5545) based configs used to run periodically
+at the given interval. Every job is allowed to have multiple schedules that can be run independently of each other.
+Don't worry, you don't need to know anything about rrule to create **rrule** based schedules. All you need to do is
+clicking some buttons over the UI. On the other hand, you should know what cron is and how to configure it to create
+**cron**-based schedules. `Cron` based examples can be found [here](#cron-schedules).
+
+Each job schedule provides different options that you can use to control the scheduling behavior of the
+[jobs command](04-Scanning.md#scheduling-jobs).
 
-## Scheduling Jobs
+#### Examples
 
-Each job may specify a `cron` compatible `schedule` to run periodically at the given interval. The `cron` format is as
-follows:
+##### RRule Schedules
+
+A schedule that runs weekly on **Friday** and scans all targets that have not yet been scanned, or
+whose last scan is older than `1 week`.
+
+![Weekly Schedules](res/weekly-schedules.png "Weekly Schedules")
+
+
+##### Cron Schedules
+
+The `cron` format is as follows:
```

Comment:
I know we had that before but I would remove cron doc here and link to some external page.